### ISSUE TITLE
`indent` rule's unexpected warnings around method chaining.

### ISSUE THREAD

**Issue by mysticatea** [eslint/eslint#4081]
I got unexpected warnings of indentations around method chaining. ESLint v1.6.0
This might be duplicate of #4076.

``` js
/* eslint-env es6 */
/* eslint indent:[2,4] */
http.get(`http://localhost:${port}/`, () => {
        reject(new Error("Already started."));
    })
    .on("error", () => {
        resolve();
    });
```

![image](https://cloud.githubusercontent.com/assets/1937871/10359323/7904a0ec-6dd2-11e5-96db-a57da5941d97.png)

I expected no errors.

I can resolve this with below:

``` js
/* eslint-env es6 */
/* eslint indent:[2,4] */
http
    .get(`http://localhost:${port}/`, () => {
        reject(new Error("Already started."));
    })
    .on("error", () => {
        resolve();
    });
```

If `http` was longer, I would do so, but....
##

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gyandeeps** [comment]
Kinda related https://github.com/eslint/eslint/issues/4073

**Comment by nzakas** [comment]
@gyandeeps is this a bug?

**Comment by gyandeeps** [comment]
I am not sure if this a bug after looking at the code.  I think this falls in the same bucket as variable declaration logic.

This should be written as 

```js
http.get(`http://localhost:${port}/`, () => {
    reject(new Error("Already started."));
})
.on("error", () => {
    resolve();
});
```

I think this sounds more like enhancement request.

**Comment by kaicataldo** [comment]
@mysticatea This has been open and unaccepted a really long time without any progress - do you think it's time to close the issue?

**Comment by mysticatea** [comment]
Hmm. `indent` rule is complicated. Each people has variety preferences.

In my opinion, `indent` rule should have the option to the 2nd line (and later) of expression statements.
My favorite is that the 2nd line (and later) of expression statements has 1 indentation. This issue is an example for that. If there is the proper issue, I don't mind closing this issue.

FWIW: Now I'm investigating token-based `indent` rule. I'm expecting that it can handle parentheses and comments by the natural way.

**Comment by kaicataldo** [comment]
@mysticatea That's very cool! I'll leave it up to you whether we close it or not - I'm not up on the latest with what's been going on with the `indent` rule. But I do think it would be nice if we could group all the `indent` rule issues into one issue/milestone.

My motivation in closing older issues is to make it easier to know what we should and shouldn't be working on :)

**Comment by ilyavolodin** [comment]
@kaicataldo That was my task (grouping all indent rules and triaging all of them). Unfortunately I have been unable to complete it yet.

**Comment by gyandeeps** [comment]
@eslint/eslint-team I think we can close this ? No movement for so long and we haven't seen any big traction for this style.

**Comment by ilyavolodin** [comment]
Closing this issue per comment above.
